examples/.cargo/config.toml

```diff
@@ -0,0 +1,31 @@
+[alias]
+esp32   = "run --features=esp32   --target=xtensa-esp32-none-elf"
+esp32c2 = "run --features=esp32c2 --target=riscv32imc-unknown-none-elf"
+esp32c3 = "run --features=esp32c3 --target=riscv32imc-unknown-none-elf"
+esp32c6 = "run --features=esp32c6 --target=riscv32imac-unknown-none-elf"
+esp32h2 = "run --features=esp32h2 --target=riscv32imac-unknown-none-elf"
+esp32p4 = "run --features=esp32p4 --target=riscv32imafc-unknown-none-elf"
+esp32s2 = "run --features=esp32s2 --target=xtensa-esp32s2-none-elf"
+esp32s3 = "run --features=esp32s3 --target=xtensa-esp32s3-none-elf"
+
+[target.'cfg(target_arch = "riscv32")']
+runner    = "espflash flash --baud=921600 --monitor"
```

Comment:
should we really set a baudrate here? there is the ESPFLASH_BAUD env variable users can/should set

Reply:
I was just using that during testing and forgot to remove it I guess haha